jbpm-designer-backend/src/main/java/org/jbpm/designer/server/EditorHandler.java

```diff
@@ -85,6 +85,8 @@
      */
     public static final String USEOLDDATAASSIGNMENTS = "designer.useolddataassignments";
 
+    public static final String SHOW_PDF_DOC = "designer.showpdfdoc";
```

Comment:
For startup system properties we do "designer.xyz". I will go over all of them to make sure they follow same standard